### PR TITLE
feat: Update docs with Helm related stuff

### PR DESCRIPTION
**Pull request by xpuska513**
## What
Update k8s docs with instructions on how to deploy airbyte using Helm charts

### CODE REVIEW

File: docs/deploying-airbyte/on-kubernetes-via-helm.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+### Install helm
+
+To install helm simply run:
+
+* For MacOS
+  * `brew install helm`
+* For Linux
+  * Download installer script `curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3`
+  * Assign required premissions `chmod 700 get_helm.sh`
+  * Run script `./get_helm.sh`
```
Comment (supertopher) [inline docs/deploying-airbyte/on-kubernetes-via-helm.md:60]: full disclosure I don't linux on the regular, but if we set perms to be `700` won't we need to `sudo ./get_helm.sh` and why not `755`.  that said this is a silly point to make

**Review by supertopher — APPROVED** [review]
@xpuska513 these are ducking fantastic docs.  nice work.  Tons of links, clear commands and expectations.  10/10 would read again.

I left alot of style nits that are subjective.  Take them or leave them.
to that end, I would add a picture even if it's silly ie [this docs page](https://docs.airbyte.com/docusaurus/contributing_to_docs) but that's even more subjective than my other nits so feel free to ignore.

File: docs/deploying-airbyte/on-kubernetes-via-helm.md (hunk 1 above)
Comment (xpuska513) [reply docs/deploying-airbyte/on-kubernetes-via-helm.md:60]: The script will ask you about sudo elevation anyways during the install process
I took the install steps from oficial helm docs
https://helm.sh/docs/intro/install/

**Comment by xpuska513** [comment]
It's still not in the finished state, since I'm going to push more updates to OSS charts soon

**Pull request merged**